src/Microsoft.SqlTools.ServiceLayer/LanguageServices/LanguageService.cs

```diff
@@ -871,12 +875,19 @@ internal SignatureHelp GetSignatureHelp(TextDocumentPosition textDocumentPositio
                                    startLine + 1,
                                    endColumn + 1,
                                    bindingContext.MetadataDisplayInfoProvider);
-                                
-                                // convert from the parser format to the VS Code wire format
-                                return AutoCompleteHelper.ConvertMethodHelpTextListToSignatureHelp(methods, 
-                                    methodLocations,
-                                    startLine + 1,
-                                    endColumn + 1);
+
+                                if (methodLocations != null)
```

Comment:
there's a null check in the method and it throws exception when it's null so when I debugging, I was getting so many exceptions because for some reason this is null many times 

---
In reply to: [91594909](https://github.com/Microsoft/sqltoolsservice/pull/167#discussion_r91594909) [](ancestors = 91594909)